src/Carbon/Lang/sk.php

```diff
@@ -74,6 +128,19 @@
         'LLL' => 'D. M. HH:mm',
         'LLLL' => 'dddd D. MMMM YYYY HH:mm',
     ],
+    'calendar' => [
+        'sameDay' => '[dnes o] LT',
+        'nextDay' => '[zajtra o] LT',
+        'lastDay' => '[včera o] LT',
+        'nextWeek' => 'dddd [o] LT',
+        'lastWeek' => function (CarbonInterface $date) {
+            return match ($date->dayOfWeek) {
+                1, 2, 4, 5 => '[minulý] dddd [o] LT', //pondelok/utorok/štvrtok/piatok
+                default => '[minulá] dddd [o] LT',
+            };
+        },
```

Comment:
We need to keep PHP 7 backward-compatibility in version 2.x (complying with semver)
```suggestion
        'lastWeek' => static function (CarbonInterface $date) {
            switch ($date->dayOfWeek) {
                case 1:
                case 2:
                case 4:
                case 5:
                    return '[minulý] dddd [o] LT'; //pondelok/utorok/štvrtok/piatok
                default:
                    return '[minulá] dddd [o] LT';
            }
        },
```